CMakeLists.txt

```diff
@@ -371,7 +370,7 @@ set_directory_properties(PROPERTIES ADDITIONAL_MAKE_CLEAN_FILES ${CMAKE_CURRENT_
 # We have added some commands to remove windows line endings and to set the exec bit
 # Reported at https://devzone.nordicsemi.com/f/nordic-q-a/47429/bug-windows-line-ending-characters
 add_custom_target(micro_eec
-	COMMAND sed -i 's/\r//g' build_all.sh WORKING_DIRECTORY ${NRF5_DIR}/external/micro-ecc
+	COMMAND sed -i 's/\\r//g' build_all.sh WORKING_DIRECTORY ${NRF5_DIR}/external/micro-ecc
```

Comment:
Actually... We can remove these manipulations altogether if we would just have proper files in https://github.com/crownstone/nRF5_SDK/tree/master/external/micro-ecc, https://github.com/crownstone/nRF5_SDK/tree/15.3.0/external/micro-ecc and https://github.com/crownstone/nRF5_SDK/tree/17.1.0/external/micro-ecc